hr_job_category/tests/test_hr_job_categories.py

```diff
@@ -108,15 +108,15 @@ def test_remove_tags_from_previous_job(self):
         """Changing the job position removes previous tags and add the new ones"""
         self.employee_id_1.write({"category_ids": self.categ_3_id})
         self.contract_id.write({"job_id": self.job_id.id})
-        self.contract_id.refresh()
+        self.contract_id.invalidate_recordset()
 
         # We have two tags (from job and the manual added one)
         self.assertEqual(len(self.employee_id_1.category_ids.ids), 2)
 
         # We change the contract of the employe
         # We should now have the tag
         self.contract_id.write({"job_id": self.job_2_id.id})
-        self.contract_id.flush()
+        self.contract_id.invalidate_recordset()
```

Comment:
`flush_recordset` perhaps? although it seems useless, we might be able to remove this line

Reply:
No, just the lines that called `flush` before